src/utils/filterTemplateProps/filterTemplateProps.spec.js

```diff
@@ -0,0 +1,20 @@
+import { expect } from 'chai'
```

Comment:
I think if you have a unit test for `pick` function, there is no need to have this one.